doc/devdocs/readme.md

```diff
@@ -80,9 +80,24 @@ modify --installpath "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\%targetFo
 - The PowerToys binaries will be in your repo under `x64\Release`.
 - If you want to copy the `PowerToys.exe` binary to a different location, you'll also need to copy the `modules` and the `svgs` folders.
 
+### Temporary build-fix for Toolset v142 (Visual Studio 2019 Version 16.8)
+
+When you use the Toolset v142 (came  typical with Visual Studio 16.8) and you cannot build the C++ project `common-md-flag` you can try one of the follow solutions.
+
+Add the two compiler switches `/await` and `/Zc:twoPhase-` to the `common-md-flag` project.
+
+![Project property change](/doc/images/v142PropertyChange.png)
+
+or add the compiler switch `/await` to the `common-md-flag` project and change the Windows SDK version of the project to version `10.1.18362.0`.
```

Comment:
Why is this getting increased.  This class is used by the interop class which then is used by others.

Reply:
1. currently `FancyZones` and the `KeyboradManger` need a higher SDK version too
2. I have no idea why this is work, I have only search for a way to make the build possible.

Reply:
KBM is a v1903 or higher item.  Just as PT Run is.

What DLL requires a higher version for FancyZones? I did a quick glance and they all are 17134 which is our min version for end users.  `Windows 10 1803 (build 17134) or later.`

Reply:
oh i see 🤦 , you are right. I currently can't build `FanceZones`, so i had assumed that it need the higher SDK too.